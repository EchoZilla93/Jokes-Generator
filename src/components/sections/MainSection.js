import React from 'react';

import Head from '../Header/Head';
import CheckBoxesRender from '../Checkbox/CheckBoxesRender';
import GetJokeBtn from '../GetJokeBtn/GetJokeBtn';
import JokeList from '../JokeList/JokeList';

export const MainSection = (props) => {
    return (
        <div className={props.className}>
            <Head />
            <CheckBoxesRender />
            <div className="getJokeBtnContainer">
                <GetJokeBtn />
                <JokeList />
            </div>
        </div>
    )
}

export default MainSection;