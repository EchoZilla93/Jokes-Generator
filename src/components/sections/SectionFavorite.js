import React from 'react';

import FavoriteCardBluePrint from '../FavoriteCard/FavoriteCardBluePrint';
import './favoriteStyles.scss'

export const SectionFavorite = ({
    isVisible }) => {
    return (
        <div className="favorite-container">
            <div
                className={`favorite-section ${isVisible ? 'favorite-section--visible' : ''}`}
            >
                <span>Favorite</span>
                <FavoriteCardBluePrint />
            </div>
        </div>
    )
}

export default SectionFavorite;