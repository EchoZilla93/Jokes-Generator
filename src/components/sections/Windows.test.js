import React from 'react';
import {
    shallow
} from 'enzyme';
import toJson from 'enzyme-to-json';

import MainSection from './MainSection';
import SectionFavorite from './SectionFavorite';

describe('windows content', () => {
    it('mainwindow renders expect snapshot', () => {
        const wrapper = shallow( < MainSection / > );
        expect(toJson(wrapper)).toMatchSnapshot();
    });
    it('favoritewindow renders expect snapshot', () => {
        const wrapper = shallow( < SectionFavorite / > );
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})