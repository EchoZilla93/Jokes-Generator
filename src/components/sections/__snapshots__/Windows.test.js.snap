// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`windows content favoritewindow renders expect snapshot 1`] = `
<div
  className="favorite-container"
>
  <div
    className="favorite-section "
  >
    <span>
      Favorite
    </span>
    <FavoriteCardBluePrint />
  </div>
</div>
`;

exports[`windows content mainwindow renders expect snapshot 1`] = `
<div>
  <Head />
  <CheckBoxRender />
  <div
    className="getJokeBtnContainer"
  >
    <GetJokeBtn />
    <JokeList />
  </div>
</div>
`;
